### PR TITLE
Pattern library integration test changes

### PR DESCRIPTION
**Pull request by shahi645**
# Motivation and Context
ROps has been updated to the latest version of pattern library. This PR is to fix the Acceptance tests against that branch

# What has changed
Few bits of refactor

# How to test?
Run with https://github.com/ONSdigital/response-operations-ui/pull/222

### CODE REVIEW

File: acceptance_tests/features/pages/breadcrumbs.py [hunk 1]
```diff
@@ -1,13 +1,16 @@
 from acceptance_tests import browser
+from selenium.common.exceptions import NoSuchElementException
 
+driver = browser.driver
```
Comment (ryangrundy7) [inline acceptance_tests/features/pages/breadcrumbs.py]: Why is `browser.driver` here?

File: acceptance_tests/features/pages/breadcrumbs.py [hunk 2]
```diff
@@ -1,13 +1,16 @@
 from acceptance_tests import browser
+from selenium.common.exceptions import NoSuchElementException
```
Comment (ryangrundy7) [inline acceptance_tests/features/pages/breadcrumbs.py]: unused import?

File: acceptance_tests/features/steps/breadcrumb_navigation.py [hunk 3]
```diff
@@ -26,6 +26,8 @@ def internal_user_cannot_see_breadcrumb_trail(_):
 @then('the user can see breadcrumbs showing the site hierarchy')
 def site_hierarchy_in_breadcrumb_trail(context):
     breadcrumbs_list = breadcrumbs.get_breadcrumbs()
+    print(breadcrumbs_list)
```
Comment (ryangrundy7) [inline acceptance_tests/features/steps/breadcrumb_navigation.py]: Don't need the print here

File: acceptance_tests/features/pages/breadcrumbs.py (hunk 1 above)
Comment (owencjones) [reply acceptance_tests/features/pages/breadcrumbs.py]: The function I'm using is only present in driver.

File: acceptance_tests/features/pages/breadcrumbs.py (hunk 2 above)
Comment (owencjones) [reply acceptance_tests/features/pages/breadcrumbs.py]: Removed.

File: acceptance_tests/features/steps/breadcrumb_navigation.py (hunk 3 above)
Comment (owencjones) [reply acceptance_tests/features/steps/breadcrumb_navigation.py]: Removed.

**Pull request closed without merging**